css/grids.css

```diff
@@ -19,36 +18,38 @@ heading2{
 	padding-left: 5%;
 	width: 100%;
 	display: grid;
-	grid-gap: 1%;
-	grid-template-columns: 2% 11% 11% 11% 11% 11% 11% 11% 11%;
-	grid-template-rows: 80px;	
+	grid-gap: 2%;
+	grid-template-columns: 0% 9% 11% 11% 11% 11% 11% 11% 11% 11%;
```

Comment:
same done here but added a percentage to create each spacing in the query of songs; also decreased the rows to match the heading